content/en/docs/concepts/sdk-configuration/exporter-selection-configuration.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "Exporter Selection Configuration"
```

Comment:
Why is it not part of `General SDK Configuration`?

Otherwise, for symmetry, it could make sense to split `General SDK Configuration` into:
- `Resource Configuration`
- `Tracing Configuration`
- `Propagation Configuration`

Reply:
Happy to pull it into general SDK configuration. I think it makes sense there.

Reply:
Wait, I did precisely what I _didn't_ want to do here. Blegh. Updating again!